pkg/kubectl/cmd/drain.go

```diff
@@ -93,6 +94,8 @@ func NewCmdUncordon(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 			cmdutil.CheckErr(options.RunCordonOrUncordon(false))
 		},
 	}
+	cmdutil.AddInclude3rdPartyFlags(cmd)
```

Comment:
Why is this line missing in NewCmdCordon?

Maybe this has been discussed already, why is this line added to almost every kubectl commands? Commands like "kubect create secret" (create_secret.go) shouldn't require knowledge of any thirdparty resource.